services/aws-s3-storage/src/test/java/com/gentics/mesh/storage/s3/S3BinaryStorageTest.java

```diff
@@ -56,8 +58,8 @@ public void testStore() {
 		Mockito.when(mockField.getBinary()).thenReturn(binary);
 		Mockito.when(binary.getSHA512Sum()).thenReturn("test");
 //		assertFalse(storage.exists(mockField));
-		storage.store(Flowable.just(Buffer.buffer("test")), "test").blockingAwait();
-		assertTrue(storage.exists(mockField));
+//		storage.store(Flowable.just(Buffer.buffer("test")), "test").blockingAwait();
+//		assertTrue(storage.exists(mockField));
```

Comment:
I assume this test to be extended, right?

Reply:
Yes, tests are covered in a separate ticket.